### ISSUE TITLE
Genevieve Genome Report not visible

### ISSUE THREAD

**Issue by madprime** [OpenHumans/open-humans#809]
## General Checkups
- [x] Have you checked that there isn't already an existing issue that describes what you report below?
- [x] Have you checked that there isn't already an open pull requests for this issue/update/change?

## Description
When I'm not logged in and load the explore/share page, I don't see Genevieve listed: https://www.openhumans.org/explore-share/

It seems it should be listed here -- users share data with it, and it currently has "approved" status.

**Comment by madprime** [comment]
This missing things is weird, but we've known this page has issues. Thinking about it, I think the explore-share page should have the following logic:

Any approved and active project that is either...
- (a) requesting access to username or data sources
- (b) NOT listed in the add-data list

This would capture a lot of what we're currently unhappy about (regarding projects not showing up). I think there's still some cases where I'm not excited by how things sort out. e.g. a survey-based research study that deposits survey data ... would only show up as "add-data".

(That said, this would hopefully be a clear improvement over the current listings.)

@gedankenstuecke what do you think?

**Comment by gedankenstuecke** [comment]
Right, the surveys are one thing, the Twitter archive would still be only 'add data', despite being designed to largely explore data. How about we give project leads a choice to choose where they'd like to be listed?

**Comment by madprime** [comment]
I like that idea! Explore/share is something we can't automatically determine.

Proposed behavior, what do you think?

* In response to "Where should your project be listed?" a project can select "Add data" and "Explore & Share", and at least one of these must be true. (Although maybe the backend should have two different fields here.)
* If "Add data" is set to "true" there must also be a "returned data description" (otherwise the form is considered invalid)
* But it _is_ allowed to have a "returned data description" without being listed as "Add data"

And for existing projects, the new fields can default according to the logic I described above.

**Comment by gedankenstuecke** [comment]
Yep, that sounds good to me! 👍

**Comment by gedankenstuecke** [comment]
Just noticed: The `Juno Exploratory` doesn't show up on either page as well.
